docs/docsite/rst/user_guide/intro_bsd.rst

```diff
@@ -49,22 +49,194 @@ Once this is done you can now use other Ansible modules apart from the ``raw`` m
 
-.. code-block:: text
+.. code-block:: ini
 
     [freebsd:vars]
     ansible_python_interpreter=/usr/local/bin/python
     [openbsd:vars]
     ansible_python_interpreter=/usr/local/bin/python3.8
 
-If you use additional plugins beyond those bundled with Ansible, you can set similar variables for ``bash``, ``perl`` or ``ruby``, depending on how the plugin is written. For example:
+
+FreeBSD packages and ports
+""""""""""""""""""""""""""
+
+In FreeBSD, there is no guarantee that either ``/usr/local/bin/python`` executable file or a link to an executable file is installed by default. The best practice for a remote host, with respect to Ansible, is to install at least the Python version supported by Ansible, e.g. ``lang/python38``, and both meta ports ``lang/python3`` and ``lang/python``. Quoting from */usr/ports/lang/python3/pkg-descr*:
+
+.. code-block:: text
+
+  This is a meta port to the Python 3.x interpreter and provides symbolic links
+  to bin/python3, bin/pydoc3, bin/idle3 and so on to allow compatibility with
+  minor version agnostic python scripts.
+
+Quoting from */usr/ports/lang/python/pkg-descr*:
+
+.. code-block:: text
+
+  This is a meta port to the Python interpreter and provides symbolic links
+  to bin/python, bin/pydoc, bin/idle and so on to allow compatibility with
+  version agnostic python scripts.
+
+As a result are installed the following packages
+
+.. code-block:: text
+
+  shell> pkg info | grep python
+  python-3.8_3,2                 "meta-port" for the default version of Python interpreter
+  python3-3_3                    Meta-port for the Python interpreter 3.x
+  python38-3.8.12_1              Interpreted object-oriented programming language
+
+and the following executables and links
+
+.. code-block:: text
+
+  shell> ll /usr/local/bin/ | grep python
+  lrwxr-xr-x  1 root  wheel       7 Jan 24 08:30 python@ -> python3
+  lrwxr-xr-x  1 root  wheel      14 Jan 24 08:30 python-config@ -> python3-config
+  lrwxr-xr-x  1 root  wheel       9 Jan 24 08:29 python3@ -> python3.8
+  lrwxr-xr-x  1 root  wheel      16 Jan 24 08:29 python3-config@ -> python3.8-config
+  -r-xr-xr-x  1 root  wheel    5248 Jan 13 01:12 python3.8*
+  -r-xr-xr-x  1 root  wheel    3153 Jan 13 01:12 python3.8-config*
+
+
+INTERPRETER_PYTHON_FALLBACK
+"""""""""""""""""""""""""""
+
+Since version 2.8 Ansible provides a useful variable ``ansible_interpreter_python_fallback`` to specify a list of paths to search for Python. See `INTERPRETER_PYTHON_FALLBACK <https://docs.ansible.com/ansible/latest/reference_appendices/config.html#interpreter-python-fallback>`_. This list will be searched and the first item found will be used. For example, the configuration below would make the installation of the meta-ports in the previous section redundant, i.e. if you don't install the Python meta ports the first two items in the list will be skipped and ``/usr/local/bin/python3.8`` will be discovered.
```

Comment:
```suggestion
Since version 2.8 Ansible provides a useful variable ``ansible_interpreter_python_fallback`` to specify a list of paths to search for Python. See :ref:`INTERPRETER_PYTHON_FALLBACK`. This list will be searched and the first item found will be used. For example, the configuration below would make the installation of the meta-ports in the previous section redundant, that is, if you don't install the Python meta ports the first two items in the list will be skipped and ``/usr/local/bin/python3.8`` will be discovered.
```